p2p/libp2p/netMessenger.go

```diff
@@ -122,12 +132,15 @@ func NewNetworkMessenger(args ArgsNetworkMessenger) (*networkMessenger, error) {
 	return p2pNode, nil
 }
 
-func checkParameters(args ArgsNetworkMessenger) error {
-	if args.Context == nil {
-		return p2p.ErrNilContext
-	}
+func setupExternalP2PLoggers() {
+	for _, external := range externalPackages {
+		logLevel := logger.GetLoggerLogLevel(fmt.Sprintf("p2p/libp2p/external/%s", external))
+		if logLevel > logger.LogTrace {
+			continue
+		}
 
-	return nil
+		_ = logging.SetLogLevel(external, "DEBUG")
```

Comment:
do we really want this directly in production code ?

Reply:
yes, we might want to debug with the help of the external p2p libs (they prove quite useful in some situations)